api/src/types.rs

```diff
@@ -153,29 +167,72 @@ pub enum OutputType {
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct Utxo {
 	/// The output commitment representing the amount
-	pub commit: pedersen::Commitment,
+	pub commit: PrintableCommitment,
```

Comment:
There is already a serializer defined for the pedersen commitment, so I had to create a small newtype struct to wrap it. Let me know if there is a better way to do this.

Reply:
There _may_ be a better way of doing this in serde but I don't know what it is...
This looks fine to me.